### DISCUSSION TITLE
Best place to keep server side helper functions/classes

### DISCUSSION THREAD

**Discussion by adityar15** [nuxt/nuxt#18979] [Questions]
I have server-side classes and helper functions. I don't want them to be included in the client bundle. If I create a helper folder inside the server folder will it do the trick?

**Comment by TheAlexLichter** [comment]
Server-side as in "only in the `/server` folder"? Then yes, use e.g. `server/utils`.
Server-side as in "only when `process.server` in Vue components"? Then I'd probably create a subfolder, e.g. `utils/server`.
